examples/experimental/i3s-17-and-debug/components/tile-validator.js

```diff
@@ -70,26 +78,128 @@ export default class TileValidator extends PureComponent {
     }
   }
 
+  _validateGeometryVsTexture(tile) {
```

Comment:
Message generation seems fairly independent of the view class... could it also be an independent global helper function in a separate file, next to tile-debug.js? 

Add JSDoc and TS?